src/renderer/components/BigSpinner.js

```diff
@@ -4,15 +4,15 @@ import React from "react";
 import styled, { keyframes, css } from "styled-components";
 
 import Box from "~/renderer/components/Box";
-import IconBigSpinner from "~/renderer/icons/BigSpinner";
+import IconBigSpinner from "~/renderer/icons/BigSpinner/index";
```

Comment:
We've got a bunch of these, isn't it the same as without the /index part?

Reply:
auto import aha (more, moving files)